lib/matplotlib/figure.py

```diff
@@ -125,12 +125,10 @@ def add(self, key, a):
         except TypeError:
             raise ValueError("first argument, %s, is not a valid key" % key)
 
-        a_existing = self.get(key)
```

Comment:
I now see that the underlying `self.get()` also casts `self._elements` through a dict.  It turns out I am deeply confused about this class (and it's super class) works.